src/GlobalStyles.js

```diff
@@ -0,0 +1,23 @@
+import { createGlobalStyle } from 'styled-components/macro'
+
+export default createGlobalStyle`
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;   
```

Comment:
You are now giving every element margin and padding of 0, is that what you had planned or was it meant to be for the body?

Reply:
It was meant for the body. Thanks for catching that!